Competition2017/src/competition/subsystems/climbing/ClimbingSubsystem.java

```diff
@@ -27,19 +30,28 @@ public ClimbingSubsystem(WPIFactory factory, XPropertyManager propManager){
         climbingMotor.setBrakeEnableDuringNeutral(true);
         climbingMotor.setControlMode(TalonControlMode.PercentVbus);
 
+        brakeSolenoid = factory.getSolenoid(3);
+        
         descendPowerProperty = propManager.createPersistentProperty("Climber descend power", -0.5);
         ascendPowerProperty = propManager.createPersistentProperty("Climber ascend power", 0.5);
     }
 
     public void stop() {
         climbingMotor.set(0);
+        brakeSolenoid.set(true);
     }
 
     public void ascend() {
         climbingMotor.set(ascendPowerProperty.get());
+        brakeSolenoid.set(false);
```

Comment:
You start the motor _and then_ disable the brake? That seems like a really bad plan.